### ISSUE TITLE
Unable to make appear check boxes

### ISSUE THREAD

**Issue by ferrufino** [softsimon/angular-2-dropdown-multiselect#24]
Hello,
So I've got this defined, since I want only one selection of an element. If one is already selected and another one is chosen I'd like to change it to that one but seems I can't. Therefore I want to show the check icon for feedback for the user. But it seems it doesn't work.

Below is how I detailed it. Please let me know if I'm missing something or it actually doesn't work.

    private myOptions: IMultiSelectOption[] = [
        { id: 1, name: 'C++' },
        { id: 2, name: 'Java' },
    ];

   private mySettings: IMultiSelectSettings = {

        selectionLimit: 1,
        closeOnSelect: true,
        checkedStyle: 'checkboxes'
    }
    private myTexts: IMultiSelectTexts = {
        checked: 'checked',
        defaultTitle: 'Programming Languages'
    };

    onChange($event){
        alert($event);
    }

called as:                     
`<ss-multiselect-dropdown [options]="myOptions" [settings]="mySettings" [texts]="myTexts" [(ngModel)]="selectedOptions" (ngModelChange)="onChange($event)"></ss-multiselect-dropdown>`

**Comment by nicholasguyett** [comment]
May I ask why you are using a multi-select dropdown if you only want the user to be able select one item at a time?  I think your use-case might be better served by either using a regular `<select>` dropdown or if its the visual aspect you're going for, a library which is designed to be a single-select dropdown.

**Comment by softsimon** [comment]
I too believe you want is a radio button or single select.

**Comment by ferrufino** [comment]
Haven't found a good alternative of a single select that works in angular 2. Bootstraps or Materialize break easily. Perhaps I'll search more for another one then.

**Comment by nicholasguyett** [comment]
I'll close this issue for now, then.

I can try to think of a library that would suit your needs if you're telling me what your goal is (that the others are not able to provide for you), if you'd like.

**Comment by ferrufino** [comment]
thanks @nicholasguyett ! Basically just a simple dropdown list, that I'm able to get the value chosen by the user and be able to toggle if a value was already selected. (For instance from my example, chose c++ and now changed it to java)

Problem seems to be that with normal dropdown list and jquery necessity, angular 2 usually breaks.

**Comment by nicholasguyett** [comment]
I personally would recommend that you use a native select item with a blank/placeholder response, rather than use a plugin for it.  It looks nice if you already have the bootstrap CSS and its guaranteed to work with Angular.  If for some reason the native `<select>` element doesn't suit your needs, I'd take a look at the (Angular UI)[https://angular-ui.github.io/bootstrap/] project.  I haven't personally used it, but its maintained and the Angular 1 compatible version was fairly populat.

In short, you should try a native select box first with a blank placeholder at the type (this requires you to initialize your property as an empty string or set both it and the placeholder's value as null, and if that doesn't suit your needs, use the Angular UI compoennts

**Comment by ferrufino** [comment]
Thanks @nicholasguyett I've tried it and it didn't work that well. Can you please tell me how to activate the checklist on this dropdown multi-select.
